app-office/gnumeric/gnumeric-1.12.52.ebuild

```diff
@@ -56,12 +56,17 @@ BDEPEND="
 	>=dev-util/intltool-0.35.0
 	virtual/pkgconfig"
 
+PATCHES=(
+	"${FILESDIR}/${P}-slibtool.patch"
+)
+
 src_prepare() {
 	default
 
 	# Manage gi overrides ourselves
 	sed '/SUBDIRS/ s/introspection//' -i Makefile.{am,in} || die
-	elibtoolize
+
+	eautoreconf
```

Comment:
Shove a comment in that this is temporary and only for the slibtool patch, once merged, we can go back to elibtoolize.

(Although, that said, that sounds kind of dodgy / not ideal given the sed above, but whatever)

Reply:
I added an comment. Personally I like going through the autoreconf process to ensure it works, but I realize others may disagree.